hello_world/api.py

```diff
@@ -17,3 +19,15 @@ def addTwoNumbersView(request):
     return HttpResponseBadRequest()
 
 
+def guest_list_endpoint(request):
+    guests = Guests.objects.all()
+    listGuest = serialize("json", guests)
+    return HttpResponse(listGuest, content_type="application/json")
+
+
+def one_guest_endpoint(request, guest_id):
+    guest = Guests.objects.filter(id=guest_id)
+    listGuest = serialize("json", guest)
```

Comment:
In this case name 'listGuest' is misleading because you have only one record instead of elements list.
Try to use eg 'jsonData' or 'guestJson'

Reply:
Corrected